client/kie-client/README.md

```diff
@@ -0,0 +1,80 @@
+## ServiceComb-Kie Client for Java
+
+   Sample Java client for ServiceComb-Kie HTTP OPENAPI. If you want more information about the ServiceComb-Kie, go [here](https://github.com/apache/servicecomb-kie).
+   
+### Build & Install
+   
+   local Build from source:
+   ```
+    maven clean install
+   ```
+   
+   add dependency to maven:
+   ```
+    <dependency>
+        <groupId>org.apache.servicecomb</groupId>
+        <artifactId>kie-client</artifactId>
+        <version>2.0.0-SNAPSHOT</version>
+    </dependency>
+   ```
+
+
+### Basic Usage
+
+#### Case 1: Create value of a key
+```
+KieClient kieClient = new KieClient();
```

Comment:
How to set up host and port?

Reply:
Thanks for your comments. I have added a case for setting host and port.